### ISSUE TITLE
Clock menu position setting from Just Perfection extension is being reseted after lock/unlock

### ISSUE THREAD

**Issue by msourada** [andia89/order-icons#11]
Hi, I’ve noticed that if I use this extension, it breaks Clock menu position setting from Just Perfection (among other things I would like to have my clock/date menu position to the far right). It works after setting it up, but when I lock/unlock screen, it resets to center. It does not happen if I disable Order Icons. It’s broken even if I try to delete date menu from the Center panel section.

Relevant issue on Just Perfection:
https://gitlab.gnome.org/jrahmatzadeh/just-perfection/-/issues/173

**Comment by msourada** [comment]
I’m on GS 43 (apart from this problem, your extension seems to work fine there).

**Comment by andia89** [comment]
Yeah there is unfortunately not sooo much i can do there, because just perfection (and other apps that change the position) are loaded after my extension. That means it kind of overrides what this extension does
